packages/js-client/src/index.test.ts

```diff
@@ -672,4 +672,3 @@ test('(Proxy) agent is not passed as request option if not set', async (t) => {
 
 const TEST_RATE_LIMIT_DELAY = 5e3
 const SECS_TO_MSECS = 1e3
-/* eslint-enable max-lines */
```

Comment:
Had to introduce this to make eslint happy. I think has part of the `lockfile` updates we may have bumped the underlying eslint version and this ended up being picked up 🤷